flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -251,10 +176,9 @@ public void createTable() {
 
     @AfterClass
     public static void closeCassandra() {
-        if (session != null) {
-            session.close();
-        }
+        session.close();
```

Comment:
Is there absolutely no way for cassandra.start() to throw an exception?

Reply:
You're right! I was too quick